cmd/job/refresh.go

```diff
@@ -163,11 +162,7 @@ func (r *refreshCommand) getRefreshDeploymentID(stream pb.JobSpecificationServic
 	for {
 		response, err := stream.Recv()
 		if err != nil {
-			if errors.Is(err, io.EOF) {
-				break
-			}
-			streamError = err
-			return
+			return "", err
```

Comment:
I remember we don't want to consider EOF as an error and hence implemented the if check right? has anything changed in our understanding?

Reply:
If yes this is the implementation followed in many places.